vmaas/webapp/vulnerabilities.py

```diff
@@ -235,7 +235,13 @@ def process_list(self, api_version, data):
                                 manually_fixable_cve_dict.setdefault(cve, {})["cve"] = cve
                                 manually_fixable_cve_dict[cve].setdefault("affected_packages", set()).add(package)
                                 # no erratum directly mappable to CVE in OVAL
-                                manually_fixable_cve_dict[cve].setdefault("errata", set())
+                                errata_ids = self.db_cache.ovaldefinition_id2errata_id.get(definition_id, [])
+                                for errata_id in errata_ids:
+                                    errata_name = self.db_cache.errataid2name.get(errata_id, None)
+                                    if errata_name is not None:
+                                        manually_fixable_cve_dict[cve].setdefault("errata", set()).add(errata_name)
+                                    else:
+                                        manually_fixable_cve_dict[cve].setdefault("errata", set())
```

Comment:
the code works but you can simplify it a bit in python with something like this
```suggestion
                                errata_names = [res for e in errata_ids if (res := self.db_cache.errataid2name.get(e))]
                                manually_fixable_cve_dict[cve].setdefault("errata", set()).update(errata_names)                                        
```

Reply:
nice :+1: